core/src/main/kotlin/com/malinskiy/marathon/execution/progress/ProgressReporter.kt

```diff
@@ -12,9 +13,10 @@ const val HUNDRED_PERCENT_IN_FLOAT: Float = 100.0f
 
 class ProgressReporter {
     private val reporters = ConcurrentHashMap<DevicePoolId, PoolProgressTracker>()
+    private lateinit var configuration: Configuration
```

Comment:
I don't appreciate to use `lateinit` without any annotation processing stuff

Reply:
I have removed this line and pass variable through constructor